Library/Homebrew/formula.rb

```diff
@@ -2880,6 +2880,7 @@ def common_stage_test_env
       GOPATH:                  "#{HOMEBREW_CACHE}/go_mod_cache",
       CARGO_HOME:              "#{HOMEBREW_CACHE}/cargo_cache",
       CURL_HOME:               ENV.fetch("CURL_HOME") { Dir.home },
+      PIP_CACHE_DIR:           "#{HOMEBREW_CACHE}/pip_cache",
```

Comment:
```suggestion
      PIP_CACHE_DIR:           "#{HOMEBREW_CACHE}/pip_cache",
      CURL_HOME:               ENV.fetch("CURL_HOME") { Dir.home },
```

I'm embarrassed that I care.